web/src/app/users/UserContactMethodEditDialogDest.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState } from 'react'
+import { useMutation, gql, CombinedError, useQuery } from 'urql'
+
+import { splitErrorsByPath } from '../util/errutil'
+import FormDialog from '../dialogs/FormDialog'
+import UserContactMethodForm, { errorPaths } from './UserContactMethodFormDest'
+import { DestinationInput } from '../../schema'
+
+type Value = {
+  name: string
+  dest: DestinationInput
+  statusUpdates: boolean
+}
+
+const query = gql`
+  query userCm($id: ID!) {
+    userContactMethod(id: $id) {
+      id
+      name
+      type
+      dest {
+        type
+        values {
+          fieldID
+          value
+          label
+        }
+      }
+      value
+      statusUpdates
+    }
+  }
+`
```

Comment:
I think we can remove type & value; we'll likely mark them as deprecated soon (in favor of only using dest)

```suggestion
const query = gql`
  query userCm($id: ID!) {
    userContactMethod(id: $id) {
      id
      name
      dest {
        type
        values {
          fieldID
          value
          label
        }
      }
      statusUpdates
    }
  }
`
```